h2o-genmodel-extensions/xgboost/src/main/java/hex/genmodel/algos/xgboost/XGBoostMojoModel.java

```diff
@@ -22,6 +48,9 @@ public XGBoostMojoModel(String[] columns, String[][] domains, String responseCol
     super(columns, domains, responseColumn);
   }
 
+  // finalize MOJO initialization after all the fields are read
+  public void postReadInit() {}
```

Comment:
Missing `@Override` ?

Reply:
This defines the method, there is no parent that would have this function.